.github/workflows/ci-nix.yml

```diff
@@ -105,7 +105,7 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           files: result/lcov.info
 
-  wasm32:
+  cross:
```

Comment:
is it stalling because of this?

Reply:
Oh, yes. Github Actions are stupid. Since you've changed it it won't see the flows it requires as passing.